### PR TITLE
zarik staging

### PR DESCRIPTION
**Pull request by zarik5**

### CODE REVIEW

File: alvr/server/src/openvr/props.rs [hunk 1]
```diff
@@ -115,7 +120,7 @@ pub extern "C" fn set_device_openvr_props(device_id: u64) {
         fn set_prop(prop: OpenvrProperty) {
             info!("Setting head OpenVR prop: {prop:?}");
             unsafe {
-                crate::SetOpenvrProperty(*HEAD_ID, to_ffi_openvr_prop(prop));
+                crate::openvr::SetOpenvrProperty(*HEAD_ID, to_ffi_openvr_prop(prop));
```
Comment (The-personified-devil) [inline alvr/server/src/openvr/props.rs:123]: Just import it too

Comment (zarik5) [reply alvr/server/src/openvr/props.rs:123]: I was a bit sloppy with imports, but i will redo them when splitting the crate in two.

File: alvr/server/src/lib.rs [hunk 2]
```diff
@@ -63,7 +49,7 @@ use std::{
 use sysinfo::{ProcessRefreshKind, RefreshKind};
 use tokio::{runtime::Runtime, sync::broadcast};
 
-static FILESYSTEM_LAYOUT: Lazy<Layout> = Lazy::new(|| {
+static FILESYSTEM_LAYOUT: Lazy<afs::Layout> = Lazy::new(|| {
```
Comment (The-personified-devil) [inline alvr/server/src/lib.rs:52]: is this even needed here anymore? It's openvr specific and also present in the openvr code

**Review by The-personified-devil — APPROVED** [review]

File: alvr/server/src/lib.rs (hunk 2 above)
Comment (zarik5) [reply alvr/server/src/lib.rs:52]: Hm. I guess this will have to be solved. But if it's ok with you we can deal with this later.

Comment (The-personified-devil) [reply alvr/server/src/lib.rs:52]: sure, later is fine

**Pull request merged**